app/navigation/Navigator.scala

```diff
@@ -28,10 +28,19 @@ import models.retrieved.RetrievedSubscription
 class Navigator @Inject()() {
 
   private val normalRoutes: Page => UserAnswers => Option[SdilReturn] => Option[RetrievedSubscription] => Option[Boolean] => Call = {
+    case PackagingSiteDetailsPage => userAnswers =>
+      sdilReturnOpt =>
+        subscriptionOpt =>
+          _ =>
+            productionSiteDetailsPageNavigation(userAnswers, sdilReturnOpt, subscriptionOpt)
```

Comment:
I think we should have a consistent line format with other lines either yours or the existing one is fine. The existing one is a bit easier to read in my humble opinion but you decide.

Reply:
done